packages/editor/src/components/editor-help/style.ios.scss

```diff
@@ -0,0 +1,6 @@
+@import "./style.scss";
+
+.button {
+	border-radius: $border-width * 8;
```

Comment:
I grabbed the border radius[ from here.](https://github.com/wordpress-mobile/WordPress-iOS/blob/try/editor-feedback/WordPress/Classes/ViewRelated/Gutenberg/Collapsable%20Header/CollapsableHeaderViewController.swift#L336)